machine/wnc_x86_64/installer.conf

```diff
@@ -0,0 +1,53 @@
+# WNC ONIE installer configuration
+
+description="WNC-X86_64"
+
+# Default ONIE block device
+install_device_platform()
+{
+    echo /dev/sda
+}
+
+# Local Variables:
+# mode: shell-script
+# eval: (sh-set-shell "/bin/sh" t nil)
+# End:
+
+#
+# Sample usage for pre/post install hooks 
+#
+pre_install()
+{
+    echo "Pre installation hook"
+
+    # 
+    # Add any platform specific hooks here
+    # before onie installation starts. 
+    #
+    # For example, maybe reset HW watchdog so
+    # the system won't reboot while onie
+    # installation is underway.
+    #
+
+    return 0
+}
+
+post_install()
```

Comment:
clearly you copied these pre/post install hooks from the kvm_x86_64.

If you do not need these hooks, then do *not* define them.  That just adds clutter.